trunk/SUAVE/Methods/Aerodynamics/AVL/write_run_cases.py

```diff
@@ -4,78 +4,77 @@
 # Created:  Dec 2014, T. Momose
 # Modified: Jan 2016, E. Botero
 #           Apr 2017, M. Clarke
+#           Aug 2019, M. Clarke
 
 # ----------------------------------------------------------------------
 #  Imports
 # ----------------------------------------------------------------------
+from SUAVE.Methods.Aerodynamics.AVL.purge_files       import purge_files
 
-from .purge_files import purge_files
-
-## @ingroup Methods-Aerodynamics-AVL
-def write_run_cases(avl_object):
+## @ingroup Analyses-AVL
+def write_run_cases(avl_object,Trim):
     """ This function writes the run cases used in the AVL batch analysis
 
     Assumptions:
         None
         
     Source:
         Drela, M. and Youngren, H., AVL, http://web.mit.edu/drela/Public/web/avl
-
+   
     Inputs:
         avl_object.current_status.batch_file                    [-]
         avl_object.geometry.mass_properties.center_of_gravity   [meters]
-
+   
     Outputs:
         None
-
+   
     Properties Used:
         N/A
-    """    
+    """       
     
 
     # unpack avl_inputs
-    batch_filename = avl_object.current_status.batch_file
     aircraft       = avl_object.geometry
+    batch_filename = avl_object.current_status.batch_file
 
     base_case_text = \
 '''
 
  ---------------------------------------------
  Run case  {0}:   {1}
 
- alpha        ->  alpha       =   {2}
- beta         ->  beta        =   {3}
+ alpha        ->  {2}       =   {3}        
```

Comment:
line this up. I hope 2 = 3 is okay

Reply:
It lined up when filled. I verified this. Check lines 115 - 140